### ISSUE TITLE
Unable to run train.py Too many errors.

### ISSUE THREAD

**Issue by Sleepyhead01** [Daffan/nav-competition-icra2022#4]
I wanted to use applr as a benchmark. However, the requirements for training are not specified. Some functions are used without defining them. For example, qt()

https://github.com/Daffan/nav-competition-icra2022/blob/8f902467016129e9756fc3d3b60735cc46466b96/applr/envs/gazebo_simulation.py#L16

Would really appreciate a walkthrough/documentation. 
Thank you in advance.

@Daffan

**Comment by Daffan** [comment]
Could you check the latest commit in the `main` branch? This should be fixed already. Thanks for your understanding!

**Comment by Daffan** [comment]
Sorry, I just realized that was in `applr` branch. We trained APPLR on a cluster, so the training pipeline does not work locally. The code for APPLR here only ensures the deployment, which means `run.py` should load APPLR policy and run the navigation. 

Here is a training pipeline that trains end-to-end deep RL policy locally. I plan to add APPLR training soon. However, the performance is not guaranteed to match branch `applr` and `e2e` at here. I will post here once the APPLR training is added.
https://github.com/Daffan/ros_jackal
